engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/ActivityTest.java

```diff
@@ -78,7 +78,7 @@ public final class ActivityTest {
                       .zeebeInput("https://github.com/{{orgId}}/{{repoId}}", "urlStatic")
                       .zeebeInputExpression(
                           "\"https://github.com/{{orgId}}/{{repoId}}\"", "urlExpression")
-                      .zeebeInput("My Name is \"Zeebe\", nice to meet you", "quotesStatic")
+                      .zeebeInput("My Name is \\\"Zeebe\\\", nice to meet you", "quotesStatic")
```

Comment:
:x: This is different from the reported issue.

In the issue, the static input was `Hello\n\nYOU!`. With this change, we escape the double quotes. I don't expect a user or the modeler to write an escaped quote. 

In the Modeler, the double quotes are encoded as `&#34;`. So, the example would be `"My Name is &#34;Zeebe&#34;, nice to meet you"`.

I suggest aligning with the modeler.